app/scripts/content/detectUI5.js

```diff
@@ -1,6 +1,17 @@
 (function () {
     'use strict';
 
+    // Create a port with background page for continuous message communication
+    var port = {
```

Comment:
The `port` object seems to be the same across files with the exception of onMessage in app\scripts\devtools\panel\ui5\main.js. Would it be possible to move `port` to `utils` and import it?